src/libraries/System.Data.OleDb/src/OleDbDataReader.cs

```diff
@@ -273,7 +273,10 @@ public override DataTable GetSchemaTable()
                 //GetSchemaTable() is defined to return null after NextResult returns false
                 //throw ADP.DataReaderNoData();
             }
+// TODO: Behavior change (https://github.com/dotnet/runtime/issues/509)
+#nullable disable
```

Comment:
Scoping this out, will submit a separate PR soon.

Reply:
Did you want to use `TODO-NULLABLE` here too?